src/UserCard.js

```diff
@@ -0,0 +1,62 @@
+import React, { useEffect, useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles({
+  root: {
+    maxWidth: 128
+  }
+});
+
+// component: semantic, variant: style
+export default function UserCard({ userId }) {
+  const [user, setUser] = useState(null);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    (async function() {
+      const res = await fetch(`https://reqres.in/api/users/${userId}`);
+      const newUser = await res.json();
+      setUser(newUser.data);
+      setIsLoading(false);
+    })();
+  }, [userId]);
+
+  const classes = useStyles();
+  return (
+    <Card>
+      {isLoading ? (
```

Comment:
This definitely works well, and is nice and easy to read. Another way to do this would be to check if we have a truth-y value for the user or not. That way, you only need to use one piece of state. 

```jsx
{ !user ? (  <Typography variant="body1" component="p">
            Loading...
          </Typography>
      ) : (
        //..render the user card
  )
}
```

Totally up to you - no right or wrong way to do it, just whatever is most readable.